manifests/repo/redhat.pp

```diff
@@ -7,18 +7,18 @@
   }
 
   yumrepo { 'Zend':
-    baseurl  => "http://repos.zend.com/zend-server/${zendserver::zend_server_version}/rpm/${::architecture}",
```

Comment:
That breaks backwards compatibility with older ZS versions. Version 7, for example: http://repos.zend.com/zend-server/7.0.0_update8/ is http://repos.zend.com/zend-server/7.0.0_update8/rpm/